python/tvm/relay/frontend/pytorch.py

```diff
@@ -132,6 +132,32 @@ def _impl(inputs, input_types):
     return _impl
 
 
+def _min_max_common(name_elemwise, name_reduce):
+    def _impl(inputs, input_types):
+        if len(inputs) == 1:
+            data = _pytorch_promote_types(inputs[:1], input_types[:1])
+            return get_relay_op(name_reduce)(data[0])
+        elif len(inputs) >= 2 and isinstance(inputs[1], int):
+            data = _pytorch_promote_types(inputs[:1], input_types[:1])
+            dim = inputs[1]
+            keepdims = inputs[2] if len(inputs) > 2 else False
+            print(dim, keepdims)
```

Comment:
I don't think we want to keep this print.

Reply:
oops, thanks. fixed